pkg/cloud/aliyunprovider.go

```diff
@@ -481,8 +481,8 @@ func (alibaba *Alibaba) NodePricing(key Key) (*Node, error) {
 
 	pricing, ok := alibaba.Pricing[keyFeature]
 	if !ok {
-		log.Warnf("Node pricing information not found for node with feature: %s", keyFeature)
-		return &Node{}, nil
+		log.Errorf("Node pricing information not found for node with feature: %s", keyFeature)
+		return nil, fmt.Errorf("Node pricing information not found for node with feature: %s letting it use default values", keyFeature)
```

Comment:
Im sending an error here because i want Opencost to calculate cost using the BaseCPUprice when error encountered. https://github.com/opencost/opencost/blob/develop/pkg/costmodel/costmodel.go#L959-L971